src/langs/en.json

```
@@ -40,6 +40,7 @@
   "PROFILE_AVATAR_CURRENT_ALT": "Profile Picture",
   "PASSWORD_TITLE": "Password",
   "PASSWORD_SETTINGS_INFORMATION": "Update your password by clicking on the 'Edit' button",
+  "NEW_PASSWORD_SETTINGS_INFORMATION": "Add a password by clicking on the 'Configure' button",
```

This string does not flow the scope convention: 
```suggestion
  "PASSWORD_SETTINGS_INFORMATION_NEW_PASSWORD": "Add a password by clicking on the 'Configure' button",
```